docs/source/installation.md

```diff
@@ -1,49 +1,69 @@
+
+To use **pyCoilGen**, you need to:
+
+* [install a supported version of Python](install-python)
+* [install the `pycoilgen` package](install-pycoilgen)
+* [install other dependencies](other-dependencies-blas-and-gfortran), if these are not already on your system.
+
+Optionally, you can install:
+
+* the [**pyCoilGen** data package](install-pycoilgen-data-package-optional)
+* [FastHenry2](fasthenry2)
+
 
 ## Install Python
 
-**pyCoilGen** depends on Python >= 3.6. It has been tested with Python 3.7.3 and 3.9.2.
+**pyCoilGen** depends on Python >= 3.6. 
 
 Please follow the instructions for your operating system to [install Python](https://www.python.org/downloads/).
 
-## Installing pyCoilGen
-
-**Note:** Ensure that you have the necessary permissions to install packages on your system. Consider using a [virtual environment](https://docs.python.org/3/library/venv.html) to manage your dependencies if necessary.
+## Install pyCoilGen
 
 To install **pyCoilGen**, you can use `pip`, the Python package manager. 
 
 ```bash
 $ pip install pycoilgen
 ```
 
-## Optional Extras
-There is an optional data package for **pyCoilGen** that provides coil mesh surface `.stl` files, pre-calculated target fields and solutions that can also be installed using `pip`.
+## Other Dependencies - BLAS and gfortran
+
+You may also need to manually install [BLAS](https://en.wikipedia.org/wiki/Basic_Linear_Algebra_Subprograms). On some Linux systems, BLAS also depends on gfortran.
 
 ```bash
-$ pip install pycoilgen_data 
+$ sudo apt-get install libopenblas-dev gfortran
 ```
 
-This will install the specified extras along with the main package. These files will automatically by detected by **pyCoilGen** after the **pyCoilGenData** package has been installed.
+## Optional packages
 
+### Install pyCoilGen Data Package
 
-## SciPiy and Dependencies
+There is an optional data package for **pyCoilGen** that provides coil mesh surface `.stl` files, pre-calculated target fields and solutions. This package can be installed with `pip`.
 
-You may need to also manually install BLAS. On some Linux systems, BLAS also depends on gfortran.
 ```bash
-$ sudo apt-get install libopenblas-dev gfortran
+$ pip install pycoilgen_data 
 ```
 
-## FastHenry2
-The `FastHenry2` application is optionally used to calculate the resistance and inductance of the coil winding. This application needs to downloaded and installed.
+These files will be automatically detected by **pyCoilGen**.
+
+
+### FastHenry2
+
+The `FastHenry2` application is used to calculate the resistance and inductance of the coil winding. This application needs to be downloaded and installed.
 
-### Windows
-Go to the [download](https://www.fastfieldsolvers.com/download.htm) page, fill out the form, then download the
+> **Note:** If this package is not installed, then the ... is not calculated. 
```

Comment:
Explain what will happen if this is not installed.

Reply:
Addressed with commit #1f11492